lib/presentation/widget/side_menu/side_menu_viewmodel.dart

```diff
@@ -40,4 +40,8 @@ class SideMenuDrawerViewModel extends BaseViewModel {
   void goToMySpace() {
     // push via nested navigation here
```

Comment:
@manhduc-nguyen Hi mate, about this you planned to add a library to redirect right ?